### PR TITLE
fix error Cannot use object of type Request as array

### PR DESCRIPTION
**Pull request by msalmikais**
I am targeting this branch, because of bugfix.

## Changelog
```markdown
### Fixed
```
## Subject
when save comment in frontend it return :
Cannot use object of type Symfony\Component\HttpFoundation\Request as array

### CODE REVIEW

**Comment by core23** [comment]
Can you please fix CS @msalmikais

**Comment by kunicmarko20** [comment]
Can you please fill in the changelog?

File: src/Controller/PostController.php [hunk 1]
```diff
@@ -293,8 +293,8 @@ public function addCommentAction($id, Request $request = null)
             ]));
         }
 
-        $form = $this->getCommentForm($post);
-        $form->submit($request);
+        $form = $this->getCommentForm($post);        
+		$form->handleRequest($request);
```
Comment (greg0ire) [inline src/Controller/PostController.php]: Please configure your IDE to use tabs, not spaces.

File: src/Controller/PostController.php [hunk 2]
```diff
@@ -293,8 +293,8 @@ public function addCommentAction($id, Request $request = null)
             ]));
         }
 
-        $form = $this->getCommentForm($post);
-        $form->submit($request);
+        $form = $this->getCommentForm($post);
```
Comment (greg0ire) [inline src/Controller/PostController.php]: What is the trailing space about?

**Pull request closed without merging**